pkg/vm/context.go

```diff
@@ -278,6 +279,21 @@ func (c *Context) IsDeployed() bool {
 	return c.NEF != nil
 }
 
+// Slot returns json formatted representation of the given slot.
+func (c *Context) Slot(n string) string {
```

Comment:
This interface always felt strange to me (the same as the one for `Stack` in VM).

Reply:
Will be fixed in https://github.com/nspcc-dev/neo-go/pull/2164.